packages/abi-typegen/src/templates/predicate/main.ts

```diff
@@ -45,7 +46,7 @@ export function renderMainTemplate(params: { abi: Abi }) {
       structs,
       enums,
       abiJsonString,
-      hexlifiedBinString,
+      compressedBytecode: compressBytecode(hexlifiedBinString),
```

Comment:
I don't want to hold progress on this one, but since the bytecode is no longer a hex string, should we consider renaming the `hexlifiedBinString` variable? It could be something else like `gzippedBase64`, `compressedBytecode`, or anything more descriptive.

Reply:
Sorry, I'm not being able to follow you here. I think the `hexlifiedBinString` variable is appropriately named as it is because it is the _uncompressed_ bytecode that we get from the `Abi` object. It is only compressed on line 49, and that variable/property on the object is named `compressedBytecode`

Reply:
Ok, now I see. Thanks.